src/server/ocpp/services/OCPPService.ts

```diff
@@ -471,10 +482,29 @@ export default class OCPPService {
       // Check
       const user = await Authorizations.isAuthorizedOnChargingStation(headers.tenantID, chargingStation,
         authorize.idTag, ServerAction.AUTHORIZE, Action.AUTHORIZE);
-      // OCPI User
-      if (user && !user.issuer && user.authorizationID) {
```

Comment:
okay, I think I get it.
I'll change it

Reply:
changed

Reply:
It should be no problem as long as the number of request is not too high.
Here is the response:

> Hi Jakob,
> 
> Since you are still in the QA/development phase it is alright if you test the AuthorizeStart/Stop using the RFID cards.
> That's the whole reason a CPO package was sent to you.
> 
> If the Sessions will stay open, it will not affect Hubject functionality in anyway because we have a session timeout duration > and eventually the Sessions will be closed.
> 
> Just be careful that you don't overload the system by sending thousands of request per day.
> 
> I am sure that will not be the case, but just a heads up.
> 
> Does this answer your questions?
> 
> 
> Best Regards,
> Mrudula Sathe